tests/utils/test_utils.py

```diff
@@ -36,13 +31,8 @@ def _check(arr, typ):
     _check(nulled, "quantitative")
     _check(["a", "b", "c"], "nominal")
 
-    if hasattr(pytest, "warns"):  # added in pytest 2.8
```

Comment:
[pytest 2.8](https://pypi.org/project/pytest/2.8.0/) was released Sep 18, 2015

Reply:
😃 Wow, some parts of this code base are old..